app/models/division.rb

```diff
@@ -50,8 +50,7 @@ def self.root_internal_name
 
   def self.root
     # create on demand if not present for convenience of blank db's and test cases
-    # @@root ||=
```

Comment:
Well, the larger structural question is whether it makes sense to create the root node in the model like that. It's meaningless-to-undesirable to do in production, and only useful for the tests. This is the sort of thing I'd prefer to see handled by other means entirely.

While it's true that `closure_tree` will return an arbitrary root node, if we're using the root as a "global parent" then there will only be one (and we could potentially write a validation or DB constraint that enforces that). That's something I'd rather do than re-define the method (which looks like a `getter`) to potentially unexpectedly _create_ a record. If that's only in there for the consideration of the tests, I think we should re-think its placement in the model.